convert/src/vertex_to_bd/badgerdoc_format/annotation_converter.py

```diff
@@ -0,0 +1,118 @@
+from typing import Any, List, Optional, Tuple
+
+from ..models import LabelStudioModel, ModelItem, ResultItem
+from ..models.bd_annotation_model import BadgerdocAnnotation, Obj, Page, Size
+from ..models.bd_tokens_model import Page as BadgerdocTokensPage
+
+LABELS = "labels"
+RELATION = "relation"
+
+
+class AnnotationConverter:
+    def convert(
+        self,
+        annotations: LabelStudioModel,
+        badgerdoc_tokens: BadgerdocTokensPage,
+    ):
+        model_items: List[ModelItem] = annotations.__root__
+        badgerdoc_annotations = BadgerdocAnnotation(
+            pages=[Page(page_num=1, size=Size(width=100, height=100))]
+        )
+        for model_item in model_items:
+            for annotation in model_item.annotations:
+                for labelstudio_item in annotation.result:
+                    self.process_labels(
+                        badgerdoc_annotations, labelstudio_item, badgerdoc_tokens
+                    )
+                for labelstudio_item in annotation.result:
+                    self.process_relations(badgerdoc_annotations, labelstudio_item)
```

Comment:
May you split this part to functions so we have less indents here?

Reply:
https://github.com/epam/badgerdoc/issues/170